cmd/api/src/queries/graph.go

```diff
@@ -585,14 +590,38 @@ func nodeToSearchResult(openGraphSearchEnabled bool, node *graph.Node) model.Sea
 	}
 }
 
-func nodesToSearchResult(openGraphSearchEnabled bool, nodes ...*graph.Node) []model.SearchResult {
-	searchResults := make([]model.SearchResult, len(nodes))
+// ETAC: use environmentsFilter to filter out environments the user does not have access to
```

Comment:
It looks a bit weird, but we'll want to keep the Go function commenting format here:
`// filterNodesToSearchResult ...`